### PR TITLE
Cis v1.2 logging network

### PR DESCRIPTION
**Pull request by ak-dassana**

### CODE REVIEW

File: content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-trail-encryption-enabled.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
+type: policy-context
+id: "cloud-trail-encryption-enabled"
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-trail-encryption-enabled.yaml:3]: Policy name, id, and file name should be the actual issue not the recommended setting.

File: content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/logs-validation-not-enabled.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
+type: policy-context
+id: logs-validation-not-enabled
```
Comment (dassana-junix) [inline content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/logs-validation-not-enabled.yaml:3]: `logs-validation-not-enabled-in-all-regions` is a better name to encapsulate both policies

**Review by dassana-junix — CHANGES_REQUESTED** [review]

File: content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-trail-encryption-not-enabled.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-trail-encryption-not-enabled.yaml:1]: Please rename file and id to be `cloudtrail-encryption-not-enabled`. Cloudtrail is a full word not needing a hypen in the middle.

File: content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-trail-encryption-not-enabled.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+type: policy-context
+id: cloud-trail-encryption-not-enabled
+name: Cloud trail encryption not enabled
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-trail-encryption-not-enabled.yaml:4]:
```suggestion
name: CloudTrail encryption not enabled
```

File: content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-watch-logs-not-enabled.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
+type: policy-context
+id: cloud-watch-logs-not-enabled
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-watch-logs-not-enabled.yaml:3]:
```suggestion
id: cloudwatch-logs-not-enabled
```

Please also rename file to not have a hypen between cloud and watch

File: content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-watch-logs-not-enabled.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+name: Cloud watch logs not enabled
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-watch-logs-not-enabled.yaml:4]:
```suggestion
name: CloudWatch logs not enabled
```

File: content/workflows/csp/aws/service/ec2/resources/route-table/policy-context/vpc-route-table-not-traffic-restricted.yaml [hunk 7]
```diff
@@ -0,0 +1,23 @@
+schema: 1.0
+type: policy-context
+id: vpc-route-table-not-traffic-restricted
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/ec2/resources/route-table/policy-context/vpc-route-table-not-traffic-restricted.yaml:3]:
```suggestion
id: vpc-route-table-is-overly-permissive
```

Please rename file as well

File: content/workflows/csp/aws/service/ec2/resources/route-table/policy-context/vpc-route-table-not-traffic-restricted.yaml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+name: VPC route table not traffic restricted
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/ec2/resources/route-table/policy-context/vpc-route-table-not-traffic-restricted.yaml:4]:
```suggestion
name: VPC route table is overly permissive
```

File: content/workflows/csp/aws/service/ec2/resources/security-group/policy-context/common-ports-are-open.yaml [hunk 9]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/ec2/resources/security-group/policy-context/common-ports-are-open.yaml:1]: We actually don't need this file. This should all be added to the existing `sg-is-wide-open` policy.

File: content/workflows/csp/aws/service/ec2/resources/security-group/policy-context/default-security-group-is-open.yaml [hunk 10]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/ec2/resources/security-group/policy-context/default-security-group-is-open.yaml:1]: We don't need this file, please add everything to the `sg-is-wide-open` policy

File: content/workflows/csp/aws/service/kms/resources/cmk/policy-context/backing-key-rotation-not-enabled.yaml [hunk 11]
```diff
@@ -0,0 +1,29 @@
+schema: 1.0
+type: policy-context
+id: backing-key-rotation-not-enabled
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/kms/resources/cmk/policy-context/backing-key-rotation-not-enabled.yaml:3]:
```suggestion
id: cmk-rotation-not-enabled
```

Please rename file as well

File: content/workflows/csp/aws/service/kms/resources/cmk/policy-context/backing-key-rotation-not-enabled.yaml [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+name: Backing key rotation not enabled
```
Comment (parth-dassana) [inline content/workflows/csp/aws/service/kms/resources/cmk/policy-context/backing-key-rotation-not-enabled.yaml:4]:
```suggestion
name: CMK rotation not enabled
```

**Review by parth-dassana — CHANGES_REQUESTED** [review]
A couple more

**Review by parth-dassana — APPROVED** [review]
LGTM, fantastic work!

**Review by parth-dassana — APPROVED** [review]
LGTM

**Pull request merged**